albumentations/augmentations/transforms.py

```diff
@@ -3240,3 +3241,115 @@ def _to_dict(self):
                 "e.g. `TemplateTransform(name='my_transform', ...)`."
             )
         return {"__class_fullname__": self.get_class_fullname(), "__name__": self.name}
+
+
+class AdvancedBlur(ImageOnlyTransform):
+    """Blur the input image using a Generalized Normal filter with a randomly selected parameters.
+        This transform also adds multiplicative noise to generated kernel before convolution.
+
+    Args:
+        blur_limit: maximum Gaussian kernel size for blurring the input image.
+            Must be zero or odd and in range [0, inf). If set to 0 it will be computed from sigma
+            as `round(sigma * (3 if img.dtype == np.uint8 else 4) * 2 + 1) + 1`.
+            If set single value `blur_limit` will be in range (0, blur_limit).
+            Default: (3, 7).
+        sigmaX_limit: Gaussian kernel standard deviation. Must be in range [0, inf).
+            If set single value `sigmaX_limit` will be in range (0, sigma_limit).
+            If set to 0 sigma will be computed as `sigma = 0.3*((ksize-1)*0.5 - 1) + 0.8`. Default: 0.
+        sigmaY_limit: Same as `sigmaY_limit` for another dimension.
+        rotate_limit: Range from which a random angle used to rotate Gaussian kernel is picked.
+            If limit is a single int an angle is picked from (-rotate_limit, rotate_limit). Default: (-90, 90).
+        beta_limit: Distribution shape parameter, 1 is the normal distribution. Default: (0.5, 8.0).
+        noise_limit: Multiplicative factor that control strength of kernel noise. Default: (0.75, 1.25).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Reference:
+        https://arxiv.org/abs/2107.10833
+
+    Targets:
+        image
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(
+        self,
+        blur_limit: Union[int, Sequence[int]] = (3, 7),
+        sigmaX_limit: Union[float, Sequence[float]] = (0.2, 1.0),
+        sigmaY_limit: Union[float, Sequence[float]] = (0.2, 1.0),
+        rotate_limit: Union[int, Sequence[int]] = 90,
```

Comment:
I think `rotate_limit` could be float

Reply:
I used `int` type here for consistency between all other augmentations that do rotations and always take integer values (but they will work for floats too).